strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py

```diff
@@ -42,6 +42,7 @@ def __init__(
         self.subscriptions: Dict[str, AsyncGenerator] = {}
         self.tasks: Dict[str, asyncio.Task] = {}
         self.completed_tasks: List[asyncio.Task] = []
+        self.connection_params: Dict[str, Any] = {}
```

Comment:
Here it's correct already :D

Reply:
To achieve consistency based on other comments, this will change to
```python
self.connection_params: Optional[Dict[str, Any]] = None`
```